x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LogicalPlanOptimizer.java

```diff
@@ -119,30 +93,6 @@ public LogicalPlanOptimizer(LogicalOptimizerContext optimizerContext) {
         super(optimizerContext);
     }
 
```

Comment:
For `LogicalPlanOptimizer`, my personal preference is to see these three+ methods in this order: `batches()`, `rules()`, `substitutions()`, `operators()`..... I find it easier to follow the logical order of rules execution.